lib/features/digital_guide_view/general_info/presentation/widgets/report_change_button.dart

```diff
@@ -1,21 +1,44 @@
+import "dart:async";
+
 import "package:flutter/material.dart";
+import "package:flutter_riverpod/flutter_riverpod.dart";
+import "package:fluttertoast/fluttertoast.dart";
 
 import "../../../../../config/ui_config.dart";
 import "../../../../../theme/app_theme.dart";
 import "../../../../../utils/context_extensions.dart";
+import "../../../../../utils/launch_url_util.dart";
 
-class ReportChangeButton extends StatelessWidget {
+class ReportChangeButton extends ConsumerWidget {
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
     return Padding(
       padding: AppWidgetsConfig.paddingMedium,
       child: Column(
         children: [
-          Text(context.localize.change_report_title),
+          Text(context.localize.report_change_title),
           const SizedBox(height: 8),
           ElevatedButton(
-            // TODO(Bartosh): handle action
-            onPressed: () {},
+            onPressed: () async {
+              final errorMessageToast =
+                  context.localize.report_change_error_toast_message;
+              final backgroundColorToast = context.colorTheme.greyLight;
+              final textColorToast = context.colorTheme.blackMirage;
+
+              if (!await ref.launch(
+                "mailto:${context.localize.report_change_email}?subject=${Uri.encodeComponent(context.localize.report_change_subject)}",
+              )) {
```

Comment:
this could be stored inside variable but i leave it up to you if u want to change it.